### DISCUSSION TITLE
How to contribute to filter lists?

### DISCUSSION THREAD

**Discussion by MagicLike** [uBlockOrigin/uAssets#17081] [Q&A] [answered]
Sorry for the dump question, but I don't want to make changes to "the wrong" file.

**Comment by iam-py-test** [comment, marked as answer]
uBo's filterlists are located at https://github.com/uBlockOrigin/uAssets/
For uBOL, these are then converted to [declarativeNetRequest rules](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/) and added to the extension package. For uBo, they are downloaded and parsed by the extension. 
A few resources about uBo syntax & filters:
- https://github.com/DandelionSprout/adfilt/blob/master/Wiki/SyntaxMeaningsThatAreActuallyHumanReadable.md
- https://github.com/gorhill/uBlock/wiki/Static-filter-syntax

Hope this is helpful.

**Reply by krystian3w** [reply]
If will there be a working tampermonkey/violentmonkey mv3 then you may have to consider a return to filters downloaded from remote servers.

**Reply by Yuki2718** [reply]
> These are then converted to [declarativeNetRequest rules](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/)

Only true to uBOL

**Reply by iam-py-test** [reply]
The OP originally posted in the section for uBOL, so I assumed they were talking about it

**Reply by uBlock-user** [reply]
He's a newbie, he doesn't need to know all the details, which will only cause him to get overwhelmed and avoid it completely. Baby steps only.

**Reply by MagicLike** [reply]
Sorry, I have just seen your answer - thank you for clarifying, I will have a look at it if I have the time! :+1:

**Comment by krystian3w** [comment]
> Sorry for the dump question

dump ≠ dumb

adds ≠ ads
